insilicho/growth_model.py

```diff
@@ -30,13 +30,47 @@ def exponential_dependence_around_optima(
     return np.exp(-((x - optima) ** 2.0) / spread**2.0)
 
 
+AUXILIARY_CONDITIONS = [
```

Comment:
What's this for? It's new, but looks unused?

Reply:
it's not used in insiliCHO (yet) but I have a need to use it in `cho_doe` (I made some modifications there). not entirely sure how to tie `AUXILIARY_CONDITIONS` into rest of insiliCHO code. Maybe I could modify `runGrowCHO._full_result.state_vars` to have a dict version of state_vars?

Reply:
hm I'd say the simplest is, if you only need it in cho_doe, just keep it there for now?

Reply:
fair point. i'll make this a docstring thing only then